roles/monitoring/tasks/cluster-logging.yaml

```diff
@@ -3,75 +3,67 @@
   file:
     path: "{{ tmpdir.path }}/logging/"
```

Comment:
On example of where `tmpdir.path` is referenced. So just wondering where this is defined now?

Reply:
https://github.com/sassoftware/viya4-deployment/blob/ansible-lint/playbooks/playbook.yaml#L3-L10